requirements.txt

```diff
@@ -5,6 +5,7 @@ git+https://gitlab.com/coala/coala-utils#egg=coala-utils-0.7.0
 colorlog>=2.7,<4.0
 git+https://gitlab.com/jayvdb/package_manager#egg=dependency-management-0.5.0.dev0
 packaging>=16.8
+pytest~=3.6.4
```

Comment:
No, why would we ever want to add pytest here?

pytest is used to run tests, and it is clearly a part of test-requirements, the requirements is specific to what is used to run coala.